ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -5,6 +5,7 @@
     catalog_location={{ item.value.location }}
     catalog_repo_url={{ item.value.url }}
     api_host={{ groups['edge'] | first }}
+    whisk_api_host={{ whisk_api_host_name | groups['edge'] | first }}
```

Comment:
let's not create a new variable, we already have 3 similar ones `whisk_api_host_name` and `whisk_api_local_host_name` and `api_host`